setup.py

```diff
@@ -33,7 +33,8 @@
     include_package_data=True,
     setup_requires=[
         "setuptools_scm<6.0.0; python_version <= '3.6'",
-        "setuptools_scm; python_version > '3.6'",
+        "setuptools_scm<8.0.0; python_version < '3.8'",
```

Comment:
```suggestion
        "setuptools_scm<8; python_version < '3.8'",
```

nit

Reply:
BTW `<= 3.6` should be replaced by `<3.7` because I think 3.6.1 would match <=3.6 which not what's intended.